dicom-archive/dicomTar.pl

```diff
@@ -212,12 +313,13 @@
 
 
 exit 0;
-# **************************************************************************************************************************  
-=pod 
-################################################
-print tarchive header
-################################################
-=cut 
+# ******************************************************************************
+=head3 archive_head()
```

Comment:
@cmadjar 
not sure if here I should have added `$finalTarget, $hostname, $system, ...)` to the list of inputs given that the function call does not pass them directly. What do you think? if yes, I can resend a separate pull request with only that change.

Reply:
I would probably not add them here since they are not passed directly.